client/src/components/Workouts/WorkoutDetail.jsx

```diff
@@ -0,0 +1,39 @@
+
+const WorkoutDetail = () => {
+    const workout = {
+        //able to add more as needed
+        title: "Sample Workout",
+        videoUrl: "https://www.youtube.com/watch?v=ErZhd3qvPYs",
+        instructions: "Go Brazy For five sets of six reps",
+        type: "Cardio",
+        muscle: "Legs",
+        difficulty: "Intermediate"
+    };
```

Comment:
`videoUrl` isn't working, Youtube wants it with the `embed` prefix. Look at this [forums](https://forum.freecodecamp.org/t/youtube-refused-to-connect/245262) for the solution. The ID remains the same

<img width="838" alt="Screenshot 2024-03-05 at 1 13 44 PM" src="https://github.com/KyoshiNoda/Elevate/assets/62672803/2150a03c-3d20-4968-8eec-10cbeaee2a2c">